android/src/main/java/com/dexterous/flutterlocalnotifications/models/NotificationDetails.java

```diff
@@ -97,6 +98,8 @@
     public static final String BODY = "body";
 
     public static final String TICKER = "ticker";
+    
+    public static final String PRECISION = "scheduledAndroidNotificationPrecision";
```

Comment:
I opted to use 'scheduledNotificationPrecision' only (except here for the key), since we're in android platform anyway

Reply:
That's fine though it I'm still seeing Android here